setup/gcp/installer/forseti_server_installer.py

```diff
@@ -92,36 +127,86 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
                 self.gcp_service_account,
                 self.user_can_grant_roles)
 
+            # Merge all the old rules if necessary
+            if self.migrate_from_v1:
+                self.merge_old_rules()
+
             # Copy the rule directory to the GCS bucket
             files.copy_file_to_destination(
                 constants.RULES_DIR_PATH, bucket_name,
                 is_directory=True, dry_run=self.config.dry_run)
 
+            self.print_copy_statement(constants.RULES_DIR_PATH, bucket_name)
```

Comment:
It might be helpful to future readers of this code to show a sample print statement example for all these banners.